docs/logs/customizing-the-sdk/README.md

```diff
@@ -30,7 +30,53 @@ TODO
 
 ### SetResourceBuilder
 
-TODO
+A `ResourceBuilder` provides a `Resource` to the `OpenTelemetryLoggerProvider`.
```

Comment:
please borrow as much from https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry#resource.

(need to test to confirm if the ENV variable is applicable to logs as well, and document accordingly)

Reply:
Do you want the full content copied here or should I link to the doc you shared?

Reply:
copy here.
(We need to later do some refactoring to extract common concepts like Resource to a common place, and have all signals refer to it.)

Also noticed that, Metrics docs doesn't talk about Resource! Good thing to add it there too.

Reply:
yeah, i noticed the other docs had TODOs for this content

Reply:
Done. I had to take out references to TracerProvider. Let me know if it needs additional edits